addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -480,6 +493,11 @@ private boolean checkMorpheusEngine() {
         return errorContains("expecting white space") || errorContains("offset");
     }
 
+    private boolean checkPgGraphqlEngine() {
+        sendQuery("query { __typename @skip(aa:tr");
+        return (errorContains("Unknown argument to @skip: aa"));
```

Comment:
More a general comment but it would help if we had the reference to the actual code, easier to double check when reviewing and for the future it there are changes.

Reply:
Do you mean adding code comments with URLs if they're available?